config/Config.cfc

```diff
@@ -13,9 +14,13 @@ component {
 
 		if ( siteTreePos ) {
 			settings.adminSideBarItems.insertAt( siteTreePos+1, "contentLibrary" );
+			settings.adminSideBarItems.insertAt( siteTreePos+2, "emailContentLibrary" );
```

Comment:
Could we use this same approach to add the alternative admin menu items?

The alt admin theme is now open source, though some/most pixl8 applications still using our internal one. Supporting the menu here will be the most portable way to do this.

Reply:
moved the menu to emailCenter